collections/_development/framework-upgrade-roadmap.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+title:  "Framework Upgrades"
+category: development
+---
+
+## Framework Upgrades
+
+There is always a clamour to utilise the latest versions of the frameworks we use to develop applications but it is important that a process is followed to ensure we don't increase the burden of supporting multiple different framework versions unnecessarily.  We should be in a position where by applications which are actively being developed are upgraded to the latest framework version as a matter of course.
+
+### New solutions
+
+New solutions should always use the latest version of the framework which is in Long Term Support.
+
+### When to upgrade
+
+* As soon as an application is undergoing significant development
+* When there is a team in place to support testing of the upgrade
+* When the new version of the framework is no longer in preview
+
+### When not to upgrade
+
+* When the new version is not covered by Long Term Support.  Details of the versions covered by LTS can be found on the [Microsoft website](https://dotnet.microsoft.com/platform/support/policy/dotnet-core)
+* When the application is dormant or changes are very minor and infrequent
+* When the application is due to be decommissioned
+
+### Upgrade process
+
+* Speak to Dev Ops - this first step in the process should always be to engage the Dev Ops team.  If you're the first person to implement the new version then additional work may be required for the Dev Ops team which will need to be planned.
+* Agree the work with your PO and DM.  If they aren't supportive then spec to either the Dev Lead, Tech Lead or Quality Guild.
```

Comment:
spelling :-)

Reply:
I love that your first git contribution in 6 months is to pick me up on my dodgy spelling. 😂

Reply:
I'm sorry I couldn't resist 😆 
Hope everything is going well.